src/components/organisms/AdminVenueView/components/Timing/Timing.tsx

```diff
@@ -32,30 +34,36 @@ export const Timing: React.FC<TimingProps> = ({
       <div className="Timing__left">
         <div className="Timing__left-bottombar">
           <div className="Timing__left-bottombar-btnleft">
-            <button
-              // href="/admin-ng"
-              className="Timing__left-bottombar-btngrey Timing__btn"
-            >
-              <FontAwesomeIcon
-                icon={faHome}
-                className="edit-button__icon"
-                size="lg"
-              />
-            </button>
+            {onClickHome && (
+              <button
+                className="Timing__left-bottombar-btngrey Timing__btn"
+                onClick={onClickHome}
+              >
+                <FontAwesomeIcon
+                  icon={faHome}
+                  className="edit-button__icon"
+                  size="lg"
+                />
+              </button>
+            )}
           </div>
           <div className="Timing__left-bottombar-btnright">
-            <button
-              className="Timing__left-bottombar-btngrey Timing__btn"
-              onClick={onClickBack}
-            >
-              Back
-            </button>
-            <button
-              className="Timing__left-bottombar-btnprimary Timing__btn"
-              onClick={onClickNext}
-            >
-              Next
-            </button>
+            {onClickBack && (
+              <button
+                className="Timing__left-bottombar-btngrey Timing__btn"
+                onClick={onClickBack}
+              >
+                Back
+              </button>
+            )}
+            {onClickNext && (
+              <button
+                className="Timing__left-bottombar-btnprimary Timing__btn"
```

Comment:
btnprimary and btngrey sound like modifiers and not elements, can you look into this BEM naming and improve it a bit?

Reply:
Open enhancement issue assigned to me. 

I didn't want to go far OOS by replacing anything with `ButtonNG`, and after https://github.com/sparkletown/sparkle/pull/1950 lands there will be prior use in the `Spaces` tab to just copy from there and improve